web/docs/Frontend Starters/Next.js/Next.js + Drupal/setting-environment-variables.md

```diff
@@ -56,3 +56,57 @@ CLIENT_SECRET
 
 See [Implementing Preview](./implementing-preview.md) for more information on
 how to implement Decoupled Preview with Next.js and Drupal.
+
+## Connecting to Multidev Environments
+
+To connect to a Multidev environment, the following helper environment variable
+can be used inside of `next.config.js`.
+
+- `IS_LIVE_ENVIRONMENT` - True if `PANTHEON_ENVIRONMENT_URL` is live.
+
+Either the `PANTHEON_CMS_ENDPOINT` or `BACKEND_URL` will need to be set.
+
+The `PANTHEON_CMS_ENDPOINT` can be mocked for local development by defining it
+in the `.env.development.local`.
+
+```
+PANTHEON_CMS_ENDPOINT=dev-my-drupal-site.pantheonsite.io
+```
+
+Taking a look at how the `next.config.js` works, there is this logic which sets
+the `backendUrl`.
+
+```
```

Comment:
Missed this on the last one, this should have the js tag so the codeblock gets syntax highlighting
```suggestion
```js
```